src/Integration/TeamExplorer/SectionController.cs

```diff
@@ -322,7 +322,7 @@ private void ExecBrowseToUrl(string url)
 
         private bool CanExecBrowseToProjectDashboard(ProjectViewModel project)
         {
-            if (project != null)
+            if (project != null && this.Host.SonarQubeService.IsConnected)
```

Comment:
Defensive coding to handle the case where the WPF command context menu asks for a command update after the server has been disconnected but before the TE UI has been reset.